compiler/literate/html.ml

```diff
@@ -158,6 +159,10 @@ let pygmentize_code (c : string Marked.pos) (language : C.backend_lang) : string
 
 (** {1 Weaving} *)
 
+let sanitize_html_href str =
+  str |> Ubase.from_utf8
```

Comment:
Wouahou I didn't know that `ubase` was a thing, actually it was released in January 2022, only after I already had coded up this piece:

https://github.com/CatalaLang/catala/blob/2da94b88c27d6017851bcd8fbffb4c7b116a5717/compiler/lcalc/backends.ml#L17-L42

Could you replace this piece of code in `backends.ml` by the use of `ubase` ? You could also get rid of the dependency on `camomile` once you do that if I remember correctly.